src/views/pages/globalping/_index.html

```diff
@@ -15,8 +15,8 @@
 
 <r-page
 	noYield="{{noYield}}"
-	title="{{title}}"
-	description="{{description}}"
```

Comment:
This is always part of the page template in case we want to customize it later. Keep it.